packages/phone-number-privacy/combiner/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@celo/phone-number-privacy-combiner",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
You might consider jumping to `1.1.11` here and keeping it in sync with the `signer` package. Very minor though and this works either way.